src/main/resources/script_templates/Filters/Bandpass.groovy

```diff
@@ -0,0 +1,57 @@
+// @OpService ops
+// @UIService ui
+// @ImgPlus img
+// @Integer radius
+// @OUTPUT ImgPlus filtered
+
+import net.imglib2.img.display.imagej.ImageJFunctions
+import net.imglib2.util.Util
+import net.imglib2.FinalDimensions
+
+// perform fft of the input
+inputFFT=ops.filter().fft(img)
+
+// display fft (by default it will be a generalized log power spectrum)
+// note that frequency 0,0 is at the origin and that the dimension in x dimensions is half that in y
+// this stack overflow question explains the reason the first transformed dimensions is half that of the second
+// http://stackoverflow.com/questions/17322449/fftw-size-of-output-array-for-2d-fourier-transform-of-3d-data
+ImageJFunctions.show(inputFFT).setTitle("fft power spectrum")
```

Comment:
It's sad that `ui.show("fft power spectrum", inputFFT)` doesn't work. I guess `UIService` should be improved to support `ComplexFloatType`.